osu.Game.Rulesets.Taiko/Mods/TaikoModRandom.cs

```diff
@@ -15,7 +15,7 @@ namespace osu.Game.Rulesets.Taiko.Mods
     public class TaikoModRandom : ModRandom, IApplicableToBeatmap
     {
         public override LocalisableString Description => @"Shuffle around the colours!";
-        public override Type[] IncompatibleMods => base.IncompatibleMods.Append(typeof(TaikoModSwap)).ToArray();
+        public override Type[] IncompatibleMods => base.IncompatibleMods.Append(typeof(TaikoModSwap)).Append(typeof(ModRelax)).ToArray();
```

Comment:
Are these changes necessary?

Reply:
Not necessary to relax diffcalc, just an afterthought.

Reply:
Copy that, and fair point. In the future if they are to be ranked, these randomisations could affect stamina values where they shouldn't for relax. I'm happy to leave these out for now as this is just a basic consideration.

Reply:
Changes reverted.